packages/@uppy/utils/src/Translator.ts

```diff
@@ -1,7 +1,21 @@
-import has from './hasProperty.js'
+import has from './hasProperty.ts'
 
-function insertReplacement (source, rx, replacement) {
-  const newParts = []
+export interface Locale<T extends number = number> {
```

Comment:
We need to document why we have this generic and how it works

Reply:
```suggestion
// We're using a generic because languages have different plural rules.
export interface Locale<T extends number = number> {
```